docs/visual_parity.md

```diff
@@ -0,0 +1,56 @@
+# Visual Parity
+
+Quilla includes a special `XPath` validation state called `VisualParity`. This page describes what `VisualParity` is in detail, how it can be used for UI regression tests, and how it integrates storage plugins to manage the images it uses and creates. Quilla strives to make the handling of auxiliary data (baseline images, etc) needed by VisualParity as simple as possible.
+
+## What is VisualParity?
```

Comment:
Please add a new enhancement to improve visual parity: Enable user to specify the xpath of an element A that should be excluded from the visual parity check of element B if A is positioned inside of B. This feature can allow users to compare two sections of a page while ignoring some parts of it that are not in the interest, for example, website publish date.

Reply:
Created issue https://github.com/microsoft/quilla/issues/54#issue-953300880